src/reactCode.jl

```diff
@@ -93,6 +93,31 @@ function het_module(Rone, Rtwo, dRone, dRtwo, hetR, hetDim, dhetDim, tr, Gas, dL
 end
 
 
+function react_module(R, dR, dLi, Gas, r, tr)
+    dRr = SVector{10}([r.binding[1]*R[1]*Gas - r.binding[2]*R[2],
+                       r.binding[3]*R[1]*Gas - r.binding[4]*R[3],
+                       r.binding[3]*R[2]*Gas - r.binding[4]*R[4],
+                       r.binding[1]*R[3]*Gas - r.binding[2]*R[4],
+                       tr.xFwd*R[1]*R[2] - r.xRev[1]*R[5],
+                       tr.xFwd*R[1]*R[3] - r.xRev[2]*R[5],
+                       tr.xFwd*R[1]*R[4] - r.xRev[3]*R[6],
+                       tr.xFwd*R[2]*R[2] - r.xRev[4]*R[6],
+                       tr.xFwd*R[3]*R[3] - r.xRev[5]*R[6],
+                       r.xFwd6*Gas*R[5] - r.xRev[6]*R[6]])
+    
+    dR[1] += - dRr[7] - dRr[6] - dRr[5] - dRr[1] - dRr[2]; # AXL
```

Comment:
Semicolons aren't necessary in Julia.

Reply:
I deleted the semicolons, but is it ok to merge with these failed checks or are these the result of some functions not being transferred over yet?

Reply:
Oh, it's just because the coverage went down. I'll override and merge.